### PR TITLE
Directory layout with a stub CI

### PR DESCRIPTION
**Pull request by j-rivero**
The PR implements a couple of things:

  * Directory structure for the python goodies
  * CI running on 3 platforms include: check-manifest to detect missing file not added to git, flake8 linter and tests under tests/

The README.md contains some instructions for basic commands of development-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# gz-mujoco converter tool
```
Comment (azeey) [inline README.md:1]: The top level directory gz-mujoco is meant to hold various things related to 
Mujoco and Gazebo. The SDFormat->MJCF conversion tool should be a subdirectory. 
See https://github.com/ignitionrobotics/gz-mujoco/pull/2.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+here = pathlib.Path(__file__).parent.resolve()
+long_description = (here / "README.md").read_text(encoding="utf-8")
+license_content = (here / "LICENSE").read_text(encoding="utf-8")
+
+
+setup(
+    name='gz_mujoco',
+    version='0.0.1',
+    description='Tool to convert SDF file to MJCF',
```
Comment (azeey) [inline setup.py:14]: nit: "SDF" -> "SDFormat"

**Review by azeey — CHANGES_REQUESTED** [review]

**Comment by azeey** [comment]
> Not sure what would be the best way of dealing with the parallel PRs in this repo. Should I make the changes here, we merge it and relocate the code in the other PRs?

Yeah, I can update the other PRs once this is merged.

File: sdformat_mjcf/tests/test_example.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import unittest
+
+from gz_mujoco.gz_mujoco import execute
```
Comment (ahcorde) [inline sdformat_mjcf/tests/test_example.py:3]:
```suggestion
from sdformat_mjcf.gz_mujoco import execute
```

File: sdformat_mjcf/tox.ini [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+[testenv]
+deps =
+    check-manifest >= 0.42
+    flake8
+commands =
+    check-manifest --ignore 'tox.ini,tests/**'
+    python setup.py check -m -s
+    flake8 .
+    python3 -m unittest {posargs}
+# python3 -m unittest {posargs}
```
Comment (ahcorde) [inline sdformat_mjcf/tox.ini:19]: remove ?

**Review by ahcorde — DISMISSED** [review]

File: sdformat_mjcf/tox.ini (hunk 4 above)
Comment (j-rivero) [reply sdformat_mjcf/tox.ini]: [f85ea26](https://github.com/ignitionrobotics/gz-mujoco/pull/3/commits/f85ea26ce9106af1f4702b8e6c3fd3d23898be72)

File: sdformat_mjcf/tests/test_example.py (hunk 3 above)
Comment (j-rivero) [reply sdformat_mjcf/tests/test_example.py]: How about [f31fa7c](https://github.com/ignitionrobotics/gz-mujoco/pull/3/commits/f31fa7c67cc08282793735a091d961196ceca10e)?

File: README.md (hunk 1 above)
Comment (j-rivero) [reply README.md]: [a323322](https://github.com/ignitionrobotics/gz-mujoco/pull/3/commits/a323322f54bbf8ac897ae88ff85b7422ee1afcd5)

**Review by azeey — APPROVED** [review]
In the interest of time, I'll merge this and make any necessary fixes in follow up PRs. Thanks @j-rivero !

File: sdformat_mjcf/tests/test_example.py [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import unittest
+
+from sdformat_mjcf.sdformat_mjcf import execute
```
[the remark this answers is not in the document]
Comment (j-rivero) [reply sdformat_mjcf/tests/test_example.py:3]: That is a good question. I was following the official python repo for packaging https://github.com/pypa/sampleproject/blob/main/tests/__init__.py . The message inside `__init__.py` does not sound to me like something critical to maintain so I would go with removing it unless it does break a working part of the workflow. For packaging, we just need to exclude the whole directory.

**Pull request merged**